docs/how-to/contribute-code.rst

```diff
@@ -22,7 +22,10 @@ coding:
 
     $ python3 -m venv venv
     $ source venv/bin/activate.sh
```

Comment:
There is no .../bin/activate.sh script in virtual environments (venv or virtualenv) - the activation script for Bash is just called .../bin/activate with no extension. (I know you didn't add/change this in this PR, but it would be good to fix this if you're already updating/fixing these instructions.)
```suggestion
    $ source venv/bin/activate
```